apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1785,6 +1785,9 @@ def _write_matrix_to_denseNDArray(
             )
             return
 
+    if isinstance(matrix, sp.csc_matrix):
+        matrix = sp.csr_matrix(matrix)
```

Comment:
This is a workaround for getting CSC matrices to write properly with the new `clib.DenseNDArray` bindings. I would appreciate it if someone could suggest a better solution.

Reply:
Thank you so much @johnkerl! This is extremely helpful, and I really appreciate it. I couldn't figure it out after a weekend of debugging.